crates/wdk-sys/src/wdf.rs

```diff
@@ -20,4 +20,4 @@ mod bindings {
     include!(concat!(env!("OUT_DIR"), "/wdf.rs"));
 }
 
-include!(concat!(env!("OUT_DIR"), "/wdf_function_table.rs"));
+include!(concat!(env!("OUT_DIR"), "/wdf_function_count.rs"));
```

Comment:
Put this inside a module named `__private`, and have the module be `#[doc(hidden)]`. I don't like our sys crate having a public api that isn't present in the WDK

Reply:
to be clear, this doesn't actually make it private. the modules still needs to be `pub` in order for the proc_macro to access it. This just hides it from the generated docs (and rust-analyzer auto-complete) and makes it clear that its an internal implementation detail.

Similar approaches are used in crates like serde to expose internal-only apis to their proc_macros: https://github.com/serde-rs/serde/blob/930401b0dd58a809fce34da091b8aa3d6083cb33/serde/src/lib.rs#L324

We also use a similar approach in our proc_macro